wurst/systems/spawns/ResourceSpawns.wurst

```diff
@@ -27,6 +32,31 @@ let islandSpawnList = new LinkedList<IslandSpawner>()
 let FISH_PER_AREA = 1
 var gameStartTime = 0.
 
+
+let spawnRegion = CreateRegion()
```

Comment:
These indents should just be the standard 4. In general, I think we should avoid any sort of formatting that doesn't indent by a standard multiple of 4.